lib/services/front.ts

```diff
@@ -187,6 +187,10 @@ export class FrontService extends Messenger implements ServiceListener, ServiceE
 						archive: false,
 					},
 					subject: details.conversation.subject,
+					// Yes, this is the best place I've found to cite the author of a message for now. SyncBot channels
+					// do not actually use there outgoing triggers anyway, so it doesn't matter *too* much.
+					// But it is still wrong.
+					to: [data.toIds.user],
```

Comment:
API docs. I haven't actually tried the API (I trust you :) ). So the docs say you can get the author back, but the API doesn't?